app/src/main/java/com/netcetera/skopjepulse/countryCitySelector/CityItem.kt

```diff
@@ -5,3 +5,16 @@ package com.netcetera.skopjepulse.countryCitySelector
  */
 
 data class CityItem (val name: String, var isChecked : Boolean = false)
```

Comment:
Think that adding `country: String` to `CityItem` is better solution. This avoids the whole `getCountryName` and the need to provide additional `countryName`  `bind(item: CountryItem, countryName: String)`
```suggestion
data class CityItem (val name: String, country: String, var isChecked : Boolean = false)
```

Reason is that having dumb views that just show what data they have instead of searching or it, it is much simpler and less bugs appear.

Reply:
Fixed: [Refactor CityItem](https://github.com/pulse-eco/pulse-eco-android/pull/5/commits/2cc8acfda4f546ae3ba1c07ed676d8b190a35b47)